src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/v2/TransactionAuthorizationOutcomeWaitingQueueConsumer.kt

```diff
@@ -0,0 +1,31 @@
+package it.pagopa.ecommerce.eventdispatcher.queues.v2
+
+import com.azure.spring.messaging.checkpoint.Checkpointer
+import io.vavr.control.Either
+import it.pagopa.ecommerce.commons.documents.v2.*
+import it.pagopa.ecommerce.commons.queues.QueueEvent
+import it.pagopa.ecommerce.eventdispatcher.queues.v2.helpers.AuthorizationRequestedEvent
+import it.pagopa.ecommerce.eventdispatcher.queues.v2.helpers.AuthorizationRequestedHelper
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import reactor.core.publisher.Mono
+
+@Service("TransactionAuthorizationOutcomeWaitingQueueConsumerV2")
+class TransactionAuthorizationOutcomeWaitingQueueConsumer(
+  @Autowired private val authorizationRequestedHelper: AuthorizationRequestedHelper,
+) {
+
+  fun messageReceiver(
+    parsedEvent:
+      Either<
+        QueueEvent<TransactionAuthorizationRequestedEvent>,
+        QueueEvent<TransactionAuthorizationOutcomeWaitingEvent>>,
```

Comment:
same as below comment, pass here only `TransactionAuthorizationOutcomeWaitingEvent` and wrap to Either.right below